make-base/docker-compose.mk

```diff
@@ -1,68 +1,45 @@
 export COMPOSE_PROJECT_NAME:=monoetna
-COMPOSE_MIXINS:=docker-compose.shared.yml $(COMPOSE_MIXINS)
 app_service_name:=${app_name}_app
-baseTag:=$(shell basename "$$(pwd)")
-fullTag:=$(IMAGES_PREFIX)$(baseTag)$(IMAGES_POSTFIX)
-baseFeTag:=${app_name}_app_fe
-fullFeTag:=$(IMAGES_PREFIX)$(baseFeTag)$(IMAGES_POSTFIX)
 containerSh:=bash
 
-docker-compose.yml:: $(wildcard ../docker/*.shared.yml) ../docker/default_compose
-	COMPOSE_MIXINS="$(COMPOSE_MIXINS)" ../docker/default_compose docker-compose.yml
-
 .PHONY: images
-images: config-ready docker-compose.yml
-	@ make -C ../docker build
 
-config-ready:: docker-compose.yml
+ifneq ($(wildcard ./docker-compose.yml),)
```

Comment:
meta magic time.  So any project that includes `docker-compose.mk` will invoke this magic: if it sees a docker-compose.yml in that directory, it will find all dependent images referenced in the docker-compose.yml file of that project, and create a target `images` which builds them.

This work is recursive -- it descends each dependent image's own dependencies correctly.